src/invidious/helpers/i18n.cr

```diff
@@ -165,3 +165,16 @@ def translate_bool(locale : String?, translation : Bool)
     return translate(locale, "No")
   end
 end
+
+def locale_is_rtl?(locale : String?)
+  # Fallback to en-US
+  return false if locale.nil?
+
+  # See https://en.wikipedia.org/wiki/Right-to-left_script#List_of_RTL_scripts
+  return true if locale == "ar" # Arabic
```

Comment:
Might be better to have a set of the rtl languages and using [includes](https://crystal-lang.org/api/1.8.0/Set.html#includes%3F%28object%29%3ABool-instance-method)?

Reply:
Ah, right!